src/Compilers/CSharp/Portable/Symbols/Source/LambdaSymbol.cs

```diff
@@ -235,6 +238,8 @@ internal Location DiagnosticLocation
             }
         }
 
+        private bool HasExplicitReturnType => (Syntax as ParenthesizedLambdaExpressionSyntax)?.ReturnType is not null;
```

Comment:
>(Syntax as ParenthesizedLambdaExpressionSyntax)?.ReturnType is not null [](http://example.com/codeflow?start=46&length=71)

Nit: could be `Syntax is ParenthesizedLambdaExpressionSyntax { ReturnType: not null }`